package.json

```diff
@@ -99,7 +100,7 @@
     "clean-css": "3.4.25",
     "css-loader": "0.28.7",
     "eslint": "5.6.1",
-    "eslint-config-jitsi": "github:jitsi/eslint-config-jitsi#1.0.0",
+    "eslint-config-jitsi": "github:jitsi/eslint-config-jitsi#5f9fe4fd00be471ab6f5d06f77ffc90bba0853e8",
```

Comment:
Make a new release please.

Reply:
Does it really make sense to make a new release every time we change a single line?

Reply:
Yes.

Reply:
can you please use the tag reference here?